### PR TITLE
`ObjectifyMorphism*` -> `CreateCapCategoryMorphism*`

### PR DESCRIPTION
**Pull request by TKuh**
Part of #1270.

### CODE REVIEW

**Comment by zickgraf** [comment]
FYI: I'm looking into the CI failures right now. The diff looks very weird.

File: CompilerForCAP/tst/loop_extraction.tst [hunk 1]
```diff
@@ -50,8 +50,8 @@ end
 
 ## inspired by UniversalMorphismIntoDirectProductWithGivenDirectProduct in SkeletalFinSets
 gap> func :=  function ( cat_1, objects_1, T_1, tau_1, P_1 )
->     return ObjectifyMorphismWithSourceAndRangeForCAPWithAttributes(
->        rec( ), cat_1,
+>     return CreateCapCategoryMorphismWithAttributes(
+         cat_1,
```
Comment (zickgraf) [inline CompilerForCAP/tst/loop_extraction.tst:54]:
```suggestion
>        cat_1,
```

That's why the diff is looking weird. Interesting that its possible to interrupt the code without causing a syntax error :D

Comment (TKuh) [reply CompilerForCAP/tst/loop_extraction.tst]: Ugh, thank you. :)

**Review by zickgraf — APPROVED** [review]
Perfect, thanks!

**Pull request merged**